md/profileCreation.md

```diff
@@ -0,0 +1,79 @@
+# Profile Editor
+
+## Definition
+
+Basically, a **profile** is an XML file. It contains the profile name, the mapping with the organization (which users / groups / roles / memberships have this profile) and the portal menu entries (menus accessible to a user logged in the portal with this profile).  
+A profile must be imported onto the portal, in the **Profiles** tab, to enable the profile (you need to connect with the **Administrator** profile to do it).  
+Before Bonita 7.6, this portal tab was the only way to create and manage profiles.  
+For development purposes, the Studio can **deploy** profiles onto the portal for you (_only for Performance and Efficiency editions_).
+
+Profiles are packaged in **files of profiles**.  
+An **file of profiles** is an XML file containing one or more profiles.  
+```xml
+<!-- file of profiles -->
+<profiles>
+	<profile>
+		<!--  First Profile -->
+	</profile>
+	<profile>
+		<!--  Second Profile -->
+	</profile>
+</profiles>
+```
+
+However, profiles in a given file of profiles are totally **independent**.  
+The purpose of grouping profiles in one file is to manage together profiles related to a same business application.  
+For example, if you want to create a leave management application for four kind of users (Employees, Managers, Human Resources officers, and Administrators) then you will have to create four applications and four profiles, and to bind each profile to the related application. As for the four applications, the four profiles should be managed together, and so grouped in the same file of profile. 
+
```

Comment:
Files of profiles are also handy to be used in different environments: you can create different files, with the same profiles, but each profile is mapped to different entities of the organization depending on the target environment.
Therefore, you could have a "profiles_dev.xml", a "profiles_UAT.xml", and a "profiles_prof.xml" files of profiles.